Client/Frontend/Browser/URLBarView.swift

```diff
@@ -903,6 +909,16 @@ extension URLBarView {
         }
     }
 
+    private func adjustStatusBar() {
```

Comment:
It would be more specific to rename the method name to smth like applyThemeOnStatusBar, adjust is too generic name for this method